src/pages/login/login.tsx

```diff
@@ -1,16 +1,113 @@
 import React from "react";
 import "./login.css";
 
+interface LoginInput {
+  email: string;
+  password: string;
+}
+
+interface ErrorFormat {
+  error: string;
+  field: string;
+}
+
+const emailRegex = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{3})+$/;
+const passwordRegex = new RegExp(
+  "^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{7,20})"
+);
+
+function validateFormInputs(input: LoginInput) {
+  if (!input.email) {
+    // return "Email is required.";
+    return { error: "Email is required.", field: "email" };
```

Comment:
Está bem claro pelo código q ele retorna "Email is required" não vejo necessidade do comentário. Vale pros outros tbm.